cli/tsc_config.rs

```diff
@@ -188,18 +188,17 @@ fn parse_compiler_options(
   Ok((value, maybe_ignored_options))
 }
 
-/// Take a string of JSONC, parse it and return a serde `Value` of the text.
+/// Take a reference to `ConfigFile`, parse `compilerOptions` and return a serde `Value` of the text.
 /// The result also contains any options that were ignored.
-pub fn parse_config(
-  config_text: &str,
-  path: &Path,
+pub fn parse_compiler_options_from_config_file(
```

Comment:
Would it make more sense to do an `.as_compiler_options()` method on the `ConfigFile` struct?

Reply:
I thought about it, but it would mean that `TsConfig` would leak into `ConfigFile`. I guess it's gonna happen sooner or later, so I'm just gonna go ahead and do that.

Reply:
Yeah, possible move TsConfig into the config as it becomes "master" place for all things config?

Reply:
As suggested I've merged `tsc_config.rs` into `config_file.rs`